functions/src/db/getRooms.ts

```diff
@@ -0,0 +1,24 @@
+import { COLLECTION_ROOMS } from './constants'
+import { UID } from '../types/uid'
+import { Room } from '../types/Room'
+import { db } from '../firebase/firebase'
+
+type Response = Pick<Room, 'id' | 'createdAt' | 'updatedAt'>
+
+export const getRooms = async (userId: UID): Promise<(Response | null)[]> => {
+    const query = await db
+        .collection(COLLECTION_ROOMS)
+        .where('uid', '==', userId)
+    //        .orderBy('createdAt')
```

Comment:
why the comment? :p

Reply:
It throw an error because i don't created index yet

Reply:
it's temporary

Reply:
you can update the https://github.com/Instant-Visio/InstantVisio-WebApp/blob/master/firestore.indexes.json so :D

Reply:
thx